app/src/main/java/com/duckduckgo/app/systemsearch/SystemSearchActivity.kt

```diff
@@ -66,6 +71,8 @@ class SystemSearchActivity : DuckDuckGoActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        Timber.i("onCreate called")
```

Comment:
Don't forget to clean this up

Reply:
Actually, I left them there intentionally since they were helpful when working on this task, but I will remove them. 👍